controller/src/display.rs

```diff
@@ -412,17 +413,17 @@ pub fn estimate(
 		bMG->"Will be locked",
 	]);
 
-	for (strategy, total, fee) in strategies {
+	for (strategy, total, fee_fields) in strategies {
 		if dark_background_color_scheme {
 			table.add_row(row![
 				bFC->strategy,
-				FR->amount_to_hr_string(fee, false),
+				FR->amount_to_hr_string(fee_fields.fee(0), false),
```

Comment:
Is it safe to use height 0 here?

Reply:
I guess hardfork4 height is better here.